tests/test_rap.py

```diff
@@ -4,15 +4,28 @@
 """
 Tests for downloading RAP model
 """
-from herbie.archive import Herbie
-from datetime import datetime
+from datetime import datetime, time
+
 import pandas as pd
+import pytest
+from dateutil import tz
 
+from herbie.archive import Herbie
+from tests.util import is_time_between
 
 today = pd.to_datetime("today").floor("1D")
 save_dir = "$TMPDIR/Herbie-Tests/"
 
 
+# Hibernate RAP AWS latest test between 00:00 and 02:30 UTC.
+now_utc = datetime.now().astimezone(tz.UTC)
+hibernate = is_time_between(now_utc, time(hour=0, minute=0), time(hour=2, minute=30))
+
+
+@pytest.mark.skipif(
+    hibernate,
+    reason="Test hibernated because upstream data not available or incomplete at night.",
+)
 def test_rap_aws():
```

Comment:
This mitigates a fluke on the test suite, happening at night time. GH-144 has more details about it.